library/dash/mdhd_contents.cc

```diff
@@ -73,7 +73,7 @@ size_t MdhdContents::Parse(const uint8_t* buffer, size_t length) {
     modification_time_ = ntohllFromBuffer(ptr);
     ptr += sizeof(uint64_t);
     timescale_ = ntohllFromBuffer(ptr);
```

Comment:
We will be pushing an update shortly that covers this change already. Line 75 should also be changed from ntohll to ntohl. You can make that adjustment then repush or wait until we push our updates in the next few weeks.